pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -211,9 +177,10 @@ def preprocess(
         * Does not support vector-Jacobian products.
         """
 
-        config = self._setup_execution_config(execution_config)
-        program = self._interface.preprocess()
-        return program, config
+        # TODO: remove comments when cuTensorNet MPS backend is available as a prototype
+        # config = self._setup_execution_config(execution_config)
+        # program = self._interface.preprocess()
+        # return program, config
```

Comment:
Can we uncomment these now? Do they need to be commented out?

Reply:
Right now, the `interface` variable cannot have a non-trivial value since we still don't have the `cutensornet` MPS backend available (or, at least, not the corresponding Python layer). If we uncomment this we have an error since the `interface` variable does not 'link' to a `preprocess()` function. We can also remove these lines as they seem odd right now, but hopefully, it will be updated soon. What do you think?